### ISSUE TITLE
Error college-recent-grads?

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [rudeboybert/fivethirtyeight#19]
There seems to be something wrong with the `college_recent_grads` file but I couldn't yet figure out where the issue is stemming from to fix it. Perhaps someone more familiar with the package can pinpoint it quicker than me?

Or if I'm making a mistake somewhere I'd appreciate a hint, thank you!

Here are my three reprexes for comparison:

1. Load the data from the package:

``` r
suppressPackageStartupMessages(library(tidyverse))
library(fivethirtyeight)

college_recent_grads %>%
  arrange(unemployment_rate) %>%
  select(major, sharewomen, unemployment_rate, sample_size, men, women) %>%
  head(5)
#> # A tibble: 5 x 6
#>                                        major sharewomen unemployment_rate
#>                                        <chr>      <dbl>             <dbl>
#> 1           Mathematics And Computer Science  0.9278072       0.000000000
#> 2                                     Botany  0.5289691       0.000000000
#> 3                               Soil Science  0.7644265       0.000000000
#> 4 Educational Administration And Supervision  0.4487323       0.000000000
#> 5  Engineering Mechanics Physics And Science  0.1839852       0.006334343
#> # ... with 3 more variables: sample_size <int>, men <int>, women <int>
```
When I saw this I didn't believe Math & CS could have 92% women, so I looked into it a bit more.

2. Load the data from the rda file in the package: I downloaded `college_recent_grads.rda` from , and ran the same code.

``` r
suppressPackageStartupMessages(library(tidyverse))
load("~/Desktop/college-grads/data/college_recent_grads.rda")

college_recent_grads %>%
  arrange(unemployment_rate) %>%
  select(major, sharewomen, unemployment_rate, sample_size, men, women) %>%
  head(5)
#> # A tibble: 5 x 6
#>                                        major sharewomen unemployment_rate
#>                                        <chr>      <dbl>             <dbl>
#> 1           Mathematics And Computer Science  0.1789819                 0
#> 2                      Military Technologies  0.0000000                 0
#> 3                                     Botany  0.5289691                 0
#> 4                               Soil Science  0.3051095                 0
#> 5 Educational Administration And Supervision  0.6517413                 0
#> # ... with 3 more variables: sample_size <int>, men <int>, women <int>
```

Note that the `sharewomen` values are different for Mathematics And Computer Science between the two outputs.

3. Load the data from 538's repo: 

``` r
suppressPackageStartupMessages(library(tidyverse))
data_from_538 <- read_csv("https://raw.githubusercontent.com/fivethirtyeight/data/master/college-majors/recent-grads.csv")
#> Parsed with column specification:
#> cols(
#>   .default = col_integer(),
#>   Major = col_character(),
#>   Major_category = col_character(),
#>   ShareWomen = col_double(),
#>   Unemployment_rate = col_double()
#> )
#> See spec(...) for full column specifications.

data_from_538 %>%
  arrange(Unemployment_rate) %>%
  select(Major, ShareWomen, Unemployment_rate, Sample_size, Men, Women) %>%
  head(5)
#> # A tibble: 5 x 6
#>                                        Major ShareWomen Unemployment_rate
#>                                        <chr>      <dbl>             <dbl>
#> 1           MATHEMATICS AND COMPUTER SCIENCE  0.1789819                 0
#> 2                      MILITARY TECHNOLOGIES  0.0000000                 0
#> 3                                     BOTANY  0.5289691                 0
#> 4                               SOIL SCIENCE  0.3051095                 0
#> 5 EDUCATIONAL ADMINISTRATION AND SUPERVISION  0.6517413                 0
#> # ... with 3 more variables: Sample_size <int>, Men <int>, Women <int>
```

This matches the .rda file but not the file loaded when the package is loaded.

**Comment by rudeboybert** [comment]
Thanks for pointing this out. We're hoping to update the package next week, so we'll look into this before then.

**Comment by mine-cetinkaya-rundel** [comment]
@rudeboybert thanks, looking forward to it. I'll use the one from the 538 repo in the meantime, those numbers seem more likely to be the right ones (sadly).

**Comment by rudeboybert** [comment]
This bug is fixed in the dev version v0.3.0.900, installable via
```
remotes::install_github("rudeboybert/fivethirtyeight")
```
We hope to release v0.4.0 on CRAN next week

**Comment by rudeboybert** [comment]
`v0.4.0` just posted to CRAN; the above error is fixed.
``` r
suppressPackageStartupMessages(library(tidyverse))
library(fivethirtyeight)
sessionInfo()
#> R version 3.4.3 (2017-11-30)
#> Platform: x86_64-apple-darwin15.6.0 (64-bit)
#> Running under: macOS High Sierra 10.13.3
#> 
#> Matrix products: default
#> BLAS: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRblas.0.dylib
#> LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib
#> 
#> locale:
#> [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#>  [1] fivethirtyeight_0.4.0 forcats_0.2.0         stringr_1.2.0        
#>  [4] dplyr_0.7.4           purrr_0.2.4           readr_1.1.1          
#>  [7] tidyr_0.8.0           tibble_1.4.2          ggplot2_2.2.1        
#> [10] tidyverse_1.2.1      
#> 
#> loaded via a namespace (and not attached):
#>  [1] Rcpp_0.12.15     cellranger_1.1.0 pillar_1.1.0     compiler_3.4.3  
#>  [5] plyr_1.8.4       bindr_0.1        tools_3.4.3      digest_0.6.15   
#>  [9] lubridate_1.7.2  jsonlite_1.5     evaluate_0.10.1  nlme_3.1-131    
#> [13] gtable_0.2.0     lattice_0.20-35  pkgconfig_2.0.1  rlang_0.1.6.9003
#> [17] psych_1.7.8      cli_1.0.0        yaml_2.1.16      parallel_3.4.3  
#> [21] haven_1.1.1      bindrcpp_0.2     xml2_1.2.0       httr_1.3.1      
#> [25] knitr_1.19       hms_0.4.1        rprojroot_1.3-2  grid_3.4.3      
#> [29] glue_1.2.0       R6_2.2.2         readxl_1.0.0     foreign_0.8-69  
#> [33] rmarkdown_1.8    modelr_0.1.1     reshape2_1.4.3   magrittr_1.5    
#> [37] backports_1.1.2  scales_0.5.0     htmltools_0.3.6  rvest_0.3.2     
#> [41] assertthat_0.2.0 mnormt_1.5-5     colorspace_1.3-2 stringi_1.1.6   
#> [45] lazyeval_0.2.1   munsell_0.4.3    broom_0.4.3      crayon_1.3.4

college_recent_grads %>%
  arrange(unemployment_rate) %>%
  select(major, sharewomen, unemployment_rate, sample_size, men, women) %>%
  head(5)
#> # A tibble: 5 x 6
#>   major                sharewomen unemployment_ra… sample_size   men women
#>   <chr>                     <dbl>            <dbl>       <int> <int> <int>
#> 1 Mathematics And Com…      0.179                0           7   500   109
#> 2 Military Technologi…      0                    0           4   124     0
#> 3 Botany                    0.529                0           9   626   703
#> 4 Soil Science              0.305                0           4   476   209
#> 5 Educational Adminis…      0.652                0           5   280   524
```

Created on 2018-02-11 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).
